systemtests/src/test/java/io/enmasse/systemtest/isolated/plans/brokered/PlansTestBrokered.java

```diff
@@ -108,4 +114,133 @@ void testReplaceAddressSpacePlanBrokered() throws Exception {
 
         getClientUtils().assertCanConnect(addressSpace, user, Arrays.asList(afterQueue, queue), resourcesManager);
     }
+
+    @Test
+    void testCreateAddressSpaceWithUnknownPlanRejected() {
+        String unknownPlan = "unknown";
+        AddressSpace addressSpace = new AddressSpaceBuilder()
+                .withNewMetadata()
+                .withName("unknown-plan-address-space")
+                .withNamespace(kubernetes.getInfraNamespace())
+                .endMetadata()
+                .withNewSpec()
+                .withType(AddressSpaceType.BROKERED.toString())
+                .withPlan(unknownPlan)
+                .endSpec()
+                .build();
+
+        try {
+            kubernetes.getAddressSpaceClient(addressSpace.getMetadata().getNamespace()).create(addressSpace);
+            fail("exception not thrown");
```

Comment:
@k-wall instead of fail and try catch use please junit5 assertThrow

Reply:
https://www.javaguides.net/2018/09/junit-5-assertthrows-example.html

Reply:
Done.